### PR TITLE
fix(lb): parse load balander id in frontend

### PR DESCRIPTION
**Pull request by Monitob**

### CODE REVIEW

File: scaleway/resource_lb_frontend.go [hunk 1]
```diff
@@ -153,14 +153,14 @@ func resourceScalewayLbFrontend() *schema.Resource {
...
+	lbAPI, zone, err := lbAPIWithZone(d, meta)
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
-	zone, lbID, err := parseZonedID(d.Get("lb_id").(string))
-	if err != nil {
-		return diag.FromErr(err)
+	lbID := expandID(d.Get("lb_id"))
+	if len(lbID) == 0 {
```
Comment (remyleone) [inline scaleway/resource_lb_frontend.go:162]: Why should it be a length list? why not compare it to an empty string?

File: scaleway/resource_lb_frontend.go [hunk 2]
```diff
@@ -211,9 +211,9 @@ func resourceScalewayLbFrontendRead(ctx context.Context, d *schema.ResourceData,
 		return diag.FromErr(err)
 	}
 
-	_, lbID, err := parseZonedID(d.Get("lb_id").(string))
-	if err != nil {
-		return diag.FromErr(err)
+	lbID := expandID(d.Get("lb_id"))
+	if len(lbID) == 0 {
```
Comment (remyleone) [inline scaleway/resource_lb_frontend.go:215]: Why should it be a length list? why not compare it to an empty string?

**Review by remyleone — CHANGES_REQUESTED** [review]

File: scaleway/resource_lb_frontend.go (hunk 1 above)
Comment (Monitob) [reply scaleway/resource_lb_frontend.go]: Just changed

File: scaleway/resource_lb_frontend.go (hunk 2 above)
Comment (Monitob) [reply scaleway/resource_lb_frontend.go]: Just changed

**Review by remyleone — APPROVED** [review]

File: scaleway/resource_lb_frontend.go [hunk 3]
```diff
@@ -153,14 +153,14 @@ func resourceScalewayLbFrontend() *schema.Resource {
...
-	lbAPI, _, err := lbAPIWithZone(d, meta)
+	lbAPI, zone, err := lbAPIWithZone(d, meta)
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
-	zone, lbID, err := parseZonedID(d.Get("lb_id").(string))
-	if err != nil {
-		return diag.FromErr(err)
+	lbID := expandID(d.Get("lb_id"))
```
Comment (remyleone) [inline scaleway/resource_lb_frontend.go:161]: What happens if the provider and the zone of id are different ?

Comment (Monitob) [reply scaleway/resource_lb_frontend.go:161]: The lb should be at the same zone

**Review by remyleone — APPROVED** [review]

**Pull request merged**